### PR TITLE
DBZ-2754 Document `log.mining.transaction.retention.hours` option

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2754

This PR documents the changes found in https://github.com/debezium/debezium-incubator/pull/251

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1581,6 +1581,10 @@ Using the default `0` will mine all archive logs.
 |The number of hours to retain entries in log mining history table.
 When set to `0`, log mining history is is disabled.
 
+|[[oracle-property-log-mining-transaction-retention-hours]]<<oracle-property-log-mining-transaction-retention-hours, `log.mining.transaction.retention.hours`>>
+|`4`
+|Positive integer value that specifies the number of hours to retain long running transactions between redo log switches.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1586]: Can you elaborate a bit more on what this exactly does? Who is doing this retention here, Debezium, or Oracle? And retain where? This warrants a bit more detailed explanation. Thanks!

**Review by gunnarmorling — CHANGES_REQUESTED** [review]

File: documentation/modules/ROOT/pages/connectors/oracle.adoc (hunk 1 above)
Comment (Naros) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1586]: I elaborated a bit more; let me know if it suffices.  Basically the property is meant to act as a soft-cap for memory usage.  If a long running transaction is kept in the in-memory transaction buffer maintained by the LogMiner implementation  for longer than the specified number of hours, then on the next log switch the transaction would be abandoned, all captured changes for the transaction discarded and no messages emitted for the transaction at all.  In short, the transaction would be skipped for all intents & purposes.

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/oracle.adoc:1586]: Ok, got it now. Can you add a comment then that we have plans to add more elaborate means of transaction buffering which will address the issue of long-running transactions more thoroughly down the road. Ideally linking to a Jira issue for reference.

**Review by gunnarmorling — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Applied, thx!

**Pull request merged**